src/Reducer/FilterReducer.js

```diff
@@ -0,0 +1,22 @@
+export function FilterReducer(state, action) {
+  const action_type = action.type.toUpperCase();
+  switch (action_type) {
+    case "ALL":
+      return { ...state, category: "ALL" };
+
+    case "ARIJIT SINGH":
```

Comment:
you can do this `Dynamically` make a `string` and push the particular `category` . this will not work if category is more than 100.

Reply:
Done..That..
Used payload also along with type..so that based on type(here category) we can update the filter state as per its payload(Arijit,KK etc.)
Thanks